maro/rl/rollout/env_sampler.py

```diff
@@ -267,6 +274,8 @@ def __init__(
         self._transition_cache: List[CacheElement] = []
         self._agent_last_index: Dict[Any, int] = {}  # Index of last occurrence of agent in self._transition_cache
         self._reward_eval_delay = reward_eval_delay
+        self._max_episode_length = max_episode_length
```

Comment:
feels like duplicated with the "max_tick", but the "truncated" info is different from "is_done". Strange

Reply:
`max_tick` is the concept of `Env` while `max_episode_length` is the concept of `EnvSampler`. A more readable name is more than welcome.